dtfs-central-api/src/v1/routes/tfm-routes.js

```diff
@@ -671,6 +671,14 @@ tfmRouter.route('/users').get(
  *         application/json:
  *           schema:
  *             $ref: '#/definitions/TFMUser'
+ *             auditDetails:
```

Comment:
One to think about, if this is the same everywhere we might want to make a definition of it and just reference that everywhere

Reply:
Its nearly the same, just has different enum values - not sure if there's a nice way to handle this without repetition

Reply:
Have decided to just do this in a verbose way as I think its neater - added to https://github.com/UK-Export-Finance/dtfs2/pull/2948 though, not here